cli/options/options.go

```diff
@@ -37,6 +37,7 @@ var RPC = []cli.Flag{
 	cli.BoolFlag{Name: "privnet, p"},
 	cli.BoolFlag{Name: "mainnet, m"},
 	cli.BoolFlag{Name: "testnet, t"},
+	cli.BoolFlag{Name: "unittest", Hidden: true},
```

Comment:
Not sure it's a good idea, hidden parameters aren't very good in general (unless of course we're talking about super cow powers).

Reply:
If we are to have `unittest` network, `Hidden` is appropriate, as it is our internal chain.
Do you want to use `privnet` instead?

Reply:
Maybe we unhide it and add some comment like "don't use it"? Technically it even exists in `netmode`, so it's not that super-secret.

Reply:
`don't use it` is exactly the case for hidden parameters. IMO `--unittest` will just overload help message.